File: Dart/que_4.dart
import 'dart:io';

int minimum(int data1, int data2) {
  if (data1 < data2) {
    return data1;
  } else {
    return data2;
  }
}

void main() {
  print("Enter the two number to find the minimum");
  int data1 = int.parse(stdin.readLineSync()!);
  int data2 = int.parse(stdin.readLineSync()!);

  int result = minimum(data1, data2);
  print("The minimum value is $result");
}

File: Python/que_4.py
# Find the minimum number

def minimum(data1, data2):
    if data1 < data2:
        return data1
    else:
        return data2

data1 = int(input("Enter the first number: "))
data2 = int(input("Enter the second number: "))

result = minimum(data1, data2)
print(f"The Minimum number is: {result}")

